src/hooks/form-value.jsx

```diff
@@ -0,0 +1,13 @@
+import { useState } from 'react';
+
+export default function useFormValue(initialValue) {
+  const [val, setVal] = useState(initialValue);
+  return [val, (data) => {
+    if (data.target) {
```

Comment:
Old entries in the database will lack the contact, so will crash, so maybe also check data && data.target?